pandas/core/config_init.py

```diff
@@ -411,6 +411,8 @@ def is_terminal() -> bool:
     True means treat None, NaN, INF, -INF as NA (old way),
     False means None and NaN are null, but INF, -INF are not NA
     (new way).
+
+.. deprecated:: 2.1.0
```

Comment:
I think this is the line that the doc build is complaining about. Maybe put it as the first line?

Reply:
hmm that didnt seem to help

Reply:
think this is handled now